packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -188,6 +191,15 @@ export function generateStack(): string | undefined {
 
     if (hasErrorInstanceId(innerError)) {
         newError.addTelemetryProperties({ innerErrorInstanceId: innerError.errorInstanceId });
+    } else {
+        newError.addTelemetryProperties({ untrustedOrigin: 1}); // To match normalizeError
```

Comment:
Hmmm this doesn't match my intention with `untrustedOrigin`.  If we wrap an error, we're articulating a new abstraction.  This is our error, the fact that it was "caused" in some sense by some untrusted error isn't important.

Or let me ask to learn your perspective - how would you plan to use this field in telemetry?